core/src/test/scala/io/aiven/guardian/kafka/MockedKafkaClientInterface.scala

```diff
@@ -47,6 +51,14 @@ class MockedKafkaClientInterface(kafkaData: Source[ReducedConsumerRecord, NotUse
   /** @return
     *   A `Sink` that allows you to commit a `CursorContext` to Kafka to signify you have processed a message
     */
-  override def commitCursor: Sink[Long, Future[Done]] = Sink.foreach(cursor => committedOffsets ++ Iterable(cursor))
```

Comment:
This change is the same as what was done in https://github.com/aiven/guardian-for-apache-kafka/pull/76, i.e. `committedOffsets ++ Iterable(cursor)` doesn't actually end up doing anything because `committedOffsets` is immutable so we are just throwing away the reference.